docs/alerts/scheduled-searches/create-real-time-alert.md

```diff
@@ -1,69 +1,69 @@
+troubleshooting timestamp discrepancies [here](/docs/send-data/collector-faq#troubleshooting-time-discrepancies).
 
-Real-Time Alerts are not duplicated, which means that if a specific raw log message has triggered an alert once already, that same log message will not trigger an alert a second time.
+Real-time alerts are not duplicated, which means that if a specific raw log message has triggered an alert once already, that same log message will not trigger an alert a second time.
 
-For example, if **Message X** caused an alert to be sent at **Time T,** and Sumo Logic detects **Message X** again at **Time T+1**, Sumo Logic does not send a second alert at **Time T+1**. But if Sumo Logic detects **Message Y** at **Time T+1**, a new alert is sent, because the root cause is different.
+For example, if **Message X** caused an alert to be sent at **Time T**, and Sumo Logic detects **Message X** again at **Time T+1**, Sumo Logic does not send a second alert at **Time T+1**. But if Sumo Logic detects **Message Y** at **Time T+1**, a new alert is sent, because the root cause is different.
 
 :::important
-If the time zone of messages is set incorrectly, those logs won't be picked up by Real-Time Alerts.
+If the time zone of messages is set incorrectly, those logs won't be picked up by real-time alerts.
 :::
 
-## Configure a Real-Time Alert
-
-To set up a Real-Time Alert:
-
-1. [Save a search](/docs/search/get-started-with-search/search-basics/save-search). 
-1. Click **Schedule this search**.<br/> ![RealTimeAlert](/img/alerts/save-real-time.png)
-1. **Run Frequency**. Select **Real Time**.
-1. For all other configuration options, see [Schedule a Search](schedule-search.md). 
-1. Click **Save**. 
 
 ## Limitations
 
-* The time range of a Real-Time Alert must be between 5 and 15 minutes. 
+* The time range of a real-time alerts must be between 5 and 15 minutes. 
 * Searching by receipt time is not supported.
-* If your search query result is a subset of your previous run's result, a Real-Time Alert will not trigger. It will trigger only when there are new results compared to the previous run.
-* A maximum of 120 emails are sent per day per Real-Time Alert.
-* Aggregate real-time Scheduled Searches evaluate the first 1,000 results per search. For Example, if the scheduled search is supposed to return more than 1,000 results, reduce the scope of the search.
-* Non-Aggregate real-time Scheduled Searches evaluate the first 100 results per search. For Example, if the scheduled search is supposed to return more than 100 results, either convert it to aggregate scheduled search or reduce the scope of the search.
-* The [`_dataTier`](/docs/manage/partitions/data-tiers) search modifier is not supported in Real-Time Alert searches.
+* If your search query result is a subset of your previous run's result, a real-time alerts will not trigger. It will trigger only when there are new results compared to the previous run.
```

Comment:
```suggestion
* If your search query result is a subset of your previous run's result, a real-time alert will not trigger. It will trigger only when there are new results compared to the previous run.
```